### ISSUE TITLE
Feature request: different colour/shape for remote branches/tags

### ISSUE THREAD

**Issue by d-torrance** [esc/git-big-picture#20]
Hello!  I recently packaged git-big-picture for Debian.

I received the following bug report (see [1]).

> Package: git-big-picture
> Severity: wishlist

> It would be useful to have git-big-picture use a different colour and or
> shape for remote branches so that they can be easily differentiated from
> local branches.
> 
> -- 
> bye,
> pabs
> 
> https://wiki.debian.org/PaulWise

[1] https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=779619

**Comment by esc** [comment]
@d-torrance yes, I have received word about the debian package already. Thanks for this! Regarding the feature request, something similar was brought up in #9
